course_material/week_05/weather_api.ipynb

```diff
@@ -20,23 +20,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 41,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "48c662d8e2fbc87db3c1601e94215d84\n"
-     ]
-    }
-   ],
```

Comment:
the text could be left with just the first 5 characters to mask it or changed to fit with the example check in my code